variables.tf

```diff
@@ -513,7 +513,7 @@ variable "nsg_name" {
 variable "egress_public_ip_name" {
   type        = string
   default     = null
-  description = "Name of pre-existing Public IP for the Network egress."
+  description = "DEPRECATED: Name of pre-existing Public IP for the Network egress."
```

Comment:
Just clarifying here. You've marked this as deprecated and left the variable in the code base so that current users of this variable will not see an error, correct? However, I don't see where this variable is actually used at this point. I would think if we had an old deprecated variable if set it would then be mapped into the new variable.

Reply:
yes, it is only left in so any code that uses it will not break.
I think what I am hearing you suggest is  that if cluster_egress_type is not set, but egress_public_ip_name IS set, then cluster_egress_type should internally be set to "userDefinedRouting". Correct? If so, I can make that change.

Reply:
Yeah, I just think if we have it their deprecated, then the user should still be able to use the value. Also might want to let folks know that the next major release of the code the variable will disappear and not be valid. Thx.